provisioning/plugins/corim-psa-decoder/test-harness/build-test-vectors.sh

```diff
@@ -24,13 +24,38 @@ TV_DOT_GO=${TV_DOT_GO?must be set in the environment.}
 
 printf "package main\n\n" > ${TV_DOT_GO}
 
-for t in ${COMID_TEMPLATES}
-do
-	cocli comid create -t ${t}.json
-	cocli corim create -m ${t}.cbor -t ${CORIM_TEMPLATE} -o corim${t}.cbor
+generate_cbor () {
+	echo "generating cbor using $1 $2"
```

Comment:
this should say something more precise, like for example "creating test vector XXX using comid YYY and corim ZZZ"